templates/site/usage.html

```diff
@@ -0,0 +1,56 @@
+{% extends "layouts/main.html" %}
+{% set page_title = "Marble Users" %}
+{% block content %}
+    {% include "partials/banner.html" %}
+
+       {% with %}
+            {% set img_src = "images/personas/persona-mark.png" %}
+            {% set img_class = "img-fluid" %}
+            {% set img_location = "left" %}
+            {% set second_img_src = "images/personas/grad-student-secondary-image-transparentbg-640.png" %}
+            {% set second_img_class = "img-fluid" %}
+            {% set second_img_location = "right" %}
+            {% set title = "Grad Students" %}
+            {% set intro_content = "" %}
+            {% set content %}
+                <p>Marble provides an all-in-one data catalog and development environment hosted by U of T or another
```

Comment:
I wouldn't single out (elevate) UofT compared to any of the other partners. Let's leave it more general.

Reply:
Removed specific mention of U of T.